### ISSUE TITLE
set up geoserver kml reflector links for CTM stakeholders

### ISSUE THREAD

**Issue by gubuntu** [kartoza/maps.kartoza.com#15]
# Problem
Some CTM stakeholders need to see some layers in Google Earth

# Proposed Solution
1. Add a CTM workspace to our main geoserver at maps.kartoza.com
1. Add a PostGIS store as a connection to the CTM postgis instance
1. Publish a couple of layers as a test
1. Style these with SLD based on existing QGIS styles
1. Test the WMS KML reflector network link in Google Earth
1. Share with Chris and get feedback
1. Pubilsh final list of layers and layer groups and give Chris the links to share

**Comment by NyakudyaA** [comment]
Added the following:
http://maps.kartoza.com/geoserver/wms/kml?layers=kartoza:five_km
http://maps.kartoza.com/geoserver/wms/kml?layers=medical_facilities

![medical](https://user-images.githubusercontent.com/2510900/58319352-0b93dd00-7e1a-11e9-8d69-967c07109324.png)

**Comment by NyakudyaA** [comment]
@gubuntu  The CTM routes layer are not rendering properly because of the structure of the data `(LineStringZM,4326)`. I believe google earth doesn't deal with that properly. All the other line features from GeoServer render fine in GE. I am going to do a quick search to see if anyone has experienced this and how they have fixed it

Eg

![eg](https://user-images.githubusercontent.com/2510900/58559599-c578b800-8223-11e9-96a7-f99ec2b2d3cd.png)

**Comment by gubuntu** [comment]
OK then you can just strip out the Z and M in a PostGIS or GeoServer SQL view

**Comment by NyakudyaA** [comment]
I added a view in the database and now we can see the layer properly in GE
![five_km](https://user-images.githubusercontent.com/2510900/58561691-f65aec00-8227-11e9-9907-fe77f88b7161.png)

**Comment by gubuntu** [comment]
So Chris just needs to give us a clear list of layers, groups and styles to set up for production use

**Comment by NyakudyaA** [comment]
Yes, I will email him and ask him to supply the list of layers he needs to share
